R/mmrm.R

```diff
@@ -248,117 +222,52 @@ fit_mmrm <- function(
             simplify = FALSE
         )
     }
-    names(params$sigma) <- levels(group)
+    names(params$sigma) <- levels(group) # TODO test this assumption is violated
```

Comment:
It seems correct.

I tried to run `mmrm` with separate covariance and `names(params$sigma)` was equal to `levels(group)`. I then tried to to re-factor `group` by changing the order of factors and then run again `mmrm`. The order of `names(params$sigma)`, as well as of the covariance matrices, was correctly inverted...

Reply:
thanks for checking, the commit I am currently working on also includes a unit test for this as well :)